rpcs3/Emu/RSX/RSXThread.h

```diff
@@ -20,7 +21,6 @@
 #include "Capture/rsx_trace.h"
 #include "Capture/rsx_replay.h"
 #include "restore_new.h"
-#include "Utilities/variant.hpp"
 #include "define_new_memleakdetect.h"
```

Comment:
define_new_memleakdetect too

Reply:
need to remove from rsxthread header as well.